superset/connectors/base/models.py

```diff
@@ -205,6 +205,7 @@ class BaseColumn(AuditMixinNullable, ImportMixin):
     min = Column(Boolean, default=False)
     filterable = Column(Boolean, default=False)
     description = Column(Text)
+    is_dttm = None
```

Comment:
The 'columns' table (used by DruidColumn) does not have that column, this will need a DB migration script.